src/pages/partySelectionContainer/partySelection/PartySelection.tsx

```diff
@@ -19,16 +19,23 @@ export default function PartySelection({ parties }: Props) {
   const { t } = useTranslation();
   const bodyTitle = t('partySelection.title');
   const bodyDescription = t('partySelection.subTitle');
-
   const [selectedParty, setSelectedParty] = React.useState<Party | null>();
   const [disableBtn, setBtnDisable] = React.useState(true);
   const history = useHistory();
   const dispatch = useAppDispatch();
 
   useEffect(() => {
-    dispatch(partiesActions.setPartySelected(undefined));
+    dispatch(
+      partiesActions.setPartySelected(
+        parties.length === 1 ? void setSelectedParty(parties[0]) : undefined
+      )
+    );
```

Comment:
this is wrong! you are changing the input to the partiesActions.setPartySelected argument to call the setSelectedParty method...

moreover, take care when use "void" we should alway catch the errors, there are some exceptions, but this time the presence of void smells :D